PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/Util.java

```diff
@@ -468,6 +473,14 @@ private static void handleDFPCustomTargetingUpdate(HashMap<String, String> bids,
         }
     }
 
+    private static void handleDFPBuilderCustomTargetingUpdate(HashMap<String, String> bids, Object adObj) {
```

Comment:
1. use `removeUsedCustomTargetingForDFP` to support `AutoRefresh` feature
2. use `addReservedKeys` to support `removeUsedCustomTargetingForDFP`

Reply:
egerger

Reply:
Hello @timrijckaert

Does this comment relevant to #199 ?

Reply:
Yes I'm sorry I couldn't comment on the other I got a 401 everytime

Reply:
thanks